src/services/subscription.service.js

```diff
@@ -0,0 +1,83 @@
+const { loggers } = require('@asymmetrik/node-fhir-server-core');
+const { v4: uuidv4 } = require('uuid');
+const db = require('../storage/DataAccess');
+const topiclist = require('../../public/topiclist.json');
+
+const logger = loggers.get('default');
+const SUBSCRIPTION = 'subscriptions';
+
+module.exports.topiclist = (_args, _context) => {
+  logger.info('Running Subsription $topic-list operation');
+  return new Promise((resolve, _reject) => {
+    resolve(topiclist);
+  });
+};
+
+module.exports.search = (_args, _context) => {
+  return new Promise((resolve, _reject) => {
+    logger.info('Subscription >>> search');
```

Comment:
Can you make sure all logger statements include an identifier? That way when we are reading through logs later we can search/filter

Reply:
So for searchById, create, update,  and remove I can use the resource id but what would the identifier be for search?

Reply:
If there are no args and it's just a "list all" operation there really is no identifier. So tbh I'm not sure logging the operation is all that useful, but it's fine as is